docs/CHANGELOG.txt

```diff
@@ -24,6 +24,72 @@ needs please refer to https://devdocs.prestashop.com/ for more information.
 Release Notes for PrestaShop 1.7
 --------------------------------
 
+####################################
+#   v1.7.7.1 - (2021-01-XX)
+####################################
+
+- Back Office:
+  - Improvement:
+    - #22020: Take shop id into account when getting a language list for an order (by @matthieu-rolland)
+    - #22028: Update out of stock mail wording (by @jolelievre)
+  - Bug fix:
+    - #21692: Refresh order products when a product is added or deleted (by @sowbiba)
+    - #22535: Method assertCmsCategoryExists doesn't return anything, it throws an exception (by @PierreRambaud)
+    - #22082: Hide product quantity in Order view page when stock management is disabled (by @jolelievre)
+    - #22299: Use OrderDetail prices when modifying an Order (by @jolelievre)
+    - #21795: Fix wrong product quantity after adding a product triggering a free gift  (by @matthieu-rolland)
+    - #22175: Added error when a customization is not found (by @Progi1984)
+    - #21912: Use language_code when locale is empty to get CLDR data (by @atomiix)
+    - #22171: When refreshing an order, display shipping price if needed (by @Progi1984)
+    - #22406: Fixed pagination in Product Quick Navigation Popup (by @Progi1984)
+    - #21977: Add debounce to quantity change of order create product (by @NeOMakinG)
+    - #22291: Fixed Pagination of Sell > Orders > Credits Slips page (by @Progi1984)
+    - #22214: Fix numeric_iso_code in the upgrade to 1.7.7.1 (by @atomiix)
+    - #22126: Fix bug on displayed price's calculation in BO order page under specific conditions (by @matthieu-rolland)
+    - #22246: Integrate ContextStateManager into refund handlers (by @jolelievre)
+    - #21568: Fixed in Create Order the update of cart rules when changing adress  (by @Progi1984)
+    - #22189: Trigger input when a predefined message is applied (by @Progi1984)
+    - #22077: [BC Break] Create Order - Fix remove cart rule when amount is not reached (by @sowbiba)
+    - #22101: Fix product selections when creating/updating tags (by @atomiix)
+    - #22099: Fix employee filter for the Logs page (by @atomiix)
+    - #22059: Add Payments block refresher - Refresh when product is added or removed from Order (by @sowbiba)
+    - #21992: Make customer note TYPE_STRING instead of TYPE_HTML (by @atomiix)
+    - #21980: Fixed the error message for a combination with a min quantity for sale (by @Progi1984)
+- Front Office:
+  - Bug fix:
+    - #22296: Center products on category list on mobile (by @NeOMakinG)
+    - #22241: Fix duplicated hook data after product page refresh (by @Hlavtox)
+    - #22103: Display combination image in product pack list (by @jolelievre)
+- Core:
+  - Bug fix:
+    - #22532: Hook keys must be in lowercase (by @PierreRambaud)
+    - #22432: Don't redirect to http from https if it is homepage (Fix Chrome security alert when customer register) (by @ludoc)
```

Comment:
```suggestion
    - #22432: Don't redirect to http from https if it is homepage (by @ludoc)
```
Suggestion (as the changelog aims to explain what has been added into the code): remove issue context, focus on the change

Reply:
Fixed